src/app/shared/log-in/x509/dynamic-x509.component.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * /users/sign-in
+ * @class LogInComponent
+ */
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'ds-dynamic-x509',
+  templateUrl: './dynamic-x509.component.html',
+  styleUrls: ['./dynamic-x509.component.scss'],
+
+})
+export class DynamicX509Component {
```

Comment:
I would not implement this component in this PR

Reply:
Deleted the x509 component form the PR